src/main/java/controller/user/LoginController.java

```diff
@@ -5,35 +5,42 @@
 import db.DataBase;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import webserver.http.Cookie;
-import webserver.http.request.Request;
-import webserver.http.response.Response;
+import webserver.http.request.HttpRequest;
+import webserver.http.response.HttpResponse;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 
+import static model.Constant.SET_COOKIE;
+
 public class LoginController extends AbstractController {
     private static final Logger logger = LoggerFactory.getLogger(LoginController.class);
 
-    public static final String ROOT_FILE = "/index.html";
     public static final String ROOT_PATH = "/";
+    public static final String ROOT_FILE = "/index.html";
     private static final String USER_LOGIN_FAIL_PATH = "/user/login_failed.html";
 
-
     @Override
-    public void doPost(Request request, Response response) {
-        logger.debug("LoginController : {}", request.getRequestPath());
+    public HttpResponse doPost(HttpRequest httpRequest) {
+        logger.debug("LoginController : {}", httpRequest.getRequestPath());
 
-        Boolean isLogin = Optional.ofNullable(DataBase.findUserById(request.getParameter("userId")))
-                .map(user -> StringUtils.equals(user.getPassword(), request.getParameter("password")))
+        Boolean isLogin = Optional.ofNullable(DataBase.findUserById(httpRequest.getParameter("userId")))
+                .map(user -> StringUtils.equals(user.getPassword(), httpRequest.getParameter("password")))
                 .orElse(false);
```

Comment:
```suggestion
                user.matchPassword(httpRequest.getParameter("password"));
```

이와 같이 User 객체에 메시지를 보내는 방식으로 구현하는 것은 어떨까?